image_processing/main.py

```diff
@@ -411,3 +412,54 @@ def upload_processed_images(bucket_name, source_file_folder):
         blob.upload_from_filename(os.path.join(source_file_folder, file_name))
 
         print("File {} uploaded to {}.".format(file_name, bucket_name))
+
+
+def trigger_generated_pictures(event, context):
+    """
+    Background Cloud Function to be triggered by Cloud Storage.
+    This generic function logs relevant data when a file is changed.
+
+    Args:
+        event (dict):  The dictionary with data specific to this type of event.
+                        The `data` field contains a description of the event in
+                        the Cloud Storage `object` format described here:
+                        https://cloud.google.com/storage/docs/json_api/v1/objects#resource
+        context (google.cloud.functions.Context): Metadata of triggering event.
+    Returns:
+        None; the output is written to Stackdriver Logging
+    """
+
+    generated_path = event["name"]
+
+    try:
+        if os.path.isfile(generated_path):
+            file_creation(generated_path)
+        else:
+            print("Error %s file not fond" % generated_path)
+    except Exception as err:
+        print('Run-time error:', err)
+
+
+def file_creation(generated_path):
+
+    # TODO(developer)
+    project_id = "your-project-id"
+    topic_id = "your-topic-id"
```

Comment:
Please fix this styling typo and re-commit your code

Reply:
is this pr still active?